doc: add tidb rollup syntax (#15053)
pingcap/docs-cn: explain-aggregation.md

5 rows in set (0.00 sec)
```

> ## 多维度数据聚合 ROLLUP

Reviewer: ```suggestion
## 多维度数据聚合 ROLLUP

```

自 v7.4.0 起，TiDB 的 `GROUP BY` 子句支持 `WITH ROLLUP` 修饰符。

> 你可以在 `GROUP BY` 子句中指定一个或多个列，形成一个分组列表，然后添加 `WITH ROLLUP` 修饰符。TiDB 将会按照分组列表中的列进行多层次的递减分组，并在输出中为你提供各个分组数据的汇总结果。

Reviewer: ```suggestion
你可以在 `GROUP BY` 子句中指定一个或多个列，形成一个分组列表，然后添加 `WITH ROLLUP` 修饰符。TiDB 将会按照分组列表中的列进行多维度的递减分组，并在输出中为你提供各个分组数据的汇总结果。
```

> > 注意：TiDB 暂时不支持 Cube 语法; TiDB 目前仅在 MPP 模式下支持 ROLLUP 语法的计划。

Reviewer: ```suggestion
> **注意**
>
> TiDB 暂不支持 Cube 语法; TiDB 目前仅在 MPP 模式下支持为 `WITH ROLLUP` 语法生成有效的执行计划。
```

|               └─Projection_16          | 3.00    | mpp[tiflash] |               | test.bank.profit, test.bank.year->Column#6, test.bank.month->Column#7                                                                                                                                                                |
|                 └─TableFullScan_17     | 3.00    | mpp[tiflash] | table:bank    | keep order:false, stats:pseudo                                                                                                                                                                                                       |
+----------------------------------------+---------+--------------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
10 rows in set (0.05 sec)
```

> 整个语句的 SQL 聚合可以在 `GROUP BY year, month WITH ROLLUP` 规范的多重分组布局 GROUPING SETS {year, month}, {year}, {} 中分别计算并连接结果。

Reviewer: ```suggestion
该语句的 SQL 聚合可以按照 `GROUP BY year, month WITH ROLLUP` 语法在  {year, month}、{year}、{} 这 3 个分组中分别计算并连接结果。
```

> 更多详情参考: [GROUP BY 修饰符](/functions-and-operators/group-by-modifier.md)

Reviewer: ```suggestion
更多信息，请参考[GROUP BY 修饰符](/functions-and-operators/group-by-modifier.md)。
```

|               └─Projection_16          | 3.00    | mpp[tiflash] |               | test.bank.profit, test.bank.year->Column#6, test.bank.month->Column#7                                                                                                                                                                |
|                 └─TableFullScan_17     | 3.00    | mpp[tiflash] | table:bank    | keep order:false, stats:pseudo                                                                                                                                                                                                       |
+----------------------------------------+---------+--------------+---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
10 rows in set (0.05 sec)
```

> 该语句的 SQL 聚合可以按照 `GROUP BY year, month WITH ROLLUP` 语法在  {year, month}、{year}、{} 这 3 个分组中分别计算并连接结果。

Reviewer: ```suggestion
该语句的 SQL 聚合可以按照 `GROUP BY year, month WITH ROLLUP` 语法在 {year, month}、{year}、{} 这 3 个分组中分别计算并连接结果。
```

> 更多信息，请参考[GROUP BY 修饰符](/functions-and-operators/group-by-modifier.md)。

Reviewer 2: ```suggestion
更多信息，请参考 [GROUP BY 修饰符](/functions-and-operators/group-by-modifier.md)。
```
